lib/keccak/keccak.c

```diff
@@ -19,6 +19,8 @@
 #define __BYTE_ORDER __LITTLE_ENDIAN
 #elif __APPLE__
 #include <machine/endian.h>
+#elif __riscv
```

Comment:
Because RISC-V is always little-endian, you can combine this with the check for "windows" above:
```
#if _MSC_VER || __riscv
```